packages/squid.rb

```diff
@@ -0,0 +1,39 @@
+require 'buildsystems/autotools'
+
+class Squid < Autotools
+  description 'Squid Web Proxy Cache'
+  homepage 'https://github.com/squid-cache/squid'
+  version '6.8'
+  license 'GPL-2.0'
+  compatibility 'all'
+  source_url 'https://www.squid-cache.org/Versions/v6/squid-6.8.tar.xz'
```

Comment:
```
source_url 'https://github.com/squid-cache/squid.git'
git_hashtag 'SQUID_6_8'
```
?

Reply:
It's faster to download tarfiles than to download from git

Reply:
A git clone is a one-time build from source and you don't need to calculate the sha256sum hash.

Reply:
My apologies, I timed both of them and downloading from git is indeed quite a bit faster.

Reply:
Good to know but there are certainly times where it is slower to clone the repo.